packages/application-shell/src/apollo-links/error-link.spec.js

```diff
@@ -66,48 +66,102 @@ describe('with unauthenticated error', () => {
           },
         ],
       },
+      unauthenticatedWithStatusCode: {
+        data: null,
+        statusCode: 401,
```

Comment:
Is this correct?

Reply:
Yeah, but we can remove it. Some APIs add a statusCode property.

Reply:
Removed.